proofs.go

```diff
@@ -162,7 +159,7 @@ func verifyProofWithUpdates(proof SparseMerkleProof, root []byte, key []byte, va
 		updates = append(updates, update)
 	}
 
-	return bytes.Equal(currentHash, root), updates
+	return bytes.Equal(currentHash, root), updates, memberValueHash
```

Comment:
What's the purpose of returning the valueHash? I don't see this output being used anywhere, except to check if it's nil in AddBranch.

Reply:
Oversight from when valueHash was used to index value - fixed